vyper/compiler.py

```diff
@@ -36,7 +31,7 @@ def mk_full_signature(code, *args, **kwargs):
     for idx, func in enumerate(abi):
         func_name = func['name'].split('(')[0]
         # Skip __init__, has no estimate
-        if func_name in abi and func_name != '__init__':
+        if func_name in gas_estimates and func_name != '__init__':
```

Comment:
What was the benefit of checking in `gas_estimates` versus `abi`? Is `gas_estimates` a filtered version of `abi`?

Reply:
@jacqueswww just answer this one an it'll be LGTM

Reply:
@fubuloubu Ha, I remember now. If you look at abi it's a list of dictionaries, func_name is a string. So obviously the a string can never be in a list of dicts :smile_cat:  So made sense to check gas_estimates which is a dict of function_names to gas usage.
Crazy one, I know!

Reply:
Also made sense to check gas_estimates considering the statement under the if.